### DISCUSSION TITLE
How to increase font size of editor chrome itself

### DISCUSSION THREAD

**Discussion by DreaminDani** [zed-industries/zed#6582] [Feature Requests]
Hi there,

I'm partially blind and need my editor controls to be at least 20% bigger than what usually comes out of the box. This is a deal breaker for me and is one of the reasons i've stuck with VS Code and WebStorm all these years.

At the very least, I'd like to be able to adjust the font size of things like the tab names, sidebar, etc. Ideally, this is just a scaling percentage, though. In VS Code, for instance, I just scale up everything to 125% and it works great.

Is there a way to do this already in Zed? I see the options to set the font_size for both the terminal and the buffer... but that's not enough for this situation.

**Comment by iamnbutler** [comment]
Hi, there isn't but we've internally been working on this on and off. There is a bit of complexity involved. Apologies this isn't available yet – No promises on timeline but we are working on it 🙂

**Reply by DreaminDani** [reply]
Ok. thanks for the quick response! Looking forward to seeing that soon :)

**Reply by charlesetc** [reply]
I just wanted to ping this. I know Zed isn't open source and I think I'm okay with that but it's getting hard for me to justify using an editor whose font size I can't change...

**Reply by JosephTLyons** [reply]
We will have some form of open source in the near future.

**Comment by JosephTLyons** [comment]
There are settings now for doing this now:

- `ui_font_size`
- `ui_font_family`
